Mathlib/Analysis/PSeries.lean

```diff
@@ -230,8 +230,8 @@ theorem summable_one_div_nat_pow {p : ℕ} :
 theorem summable_one_div_int_pow {p : ℕ} :
     (Summable fun n : ℤ => 1 / (n : ℝ) ^ p) ↔ 1 < p := by
   refine'
-    ⟨fun h => summable_one_div_nat_pow.mp (h.comp_injective Nat.cast_injective), fun h =>
-      summable_int_of_summable_nat (summable_one_div_nat_pow.mpr h)
+    ⟨fun h ↦ summable_one_div_nat_pow.mp (h.comp_injective Nat.cast_injective),
+     fun h ↦ (summable_one_div_nat_pow.mpr h).of_natCast_neg_natCast
         (((summable_one_div_nat_pow.mpr h).mul_left <| 1 / (-1 : ℝ) ^ p).congr fun n => _)⟩
```

Comment:
Why not change this to `refine` and at the end `fun n ↦ ?_` while you are at it?